docs/resources/generated-error-pages/error-codes-inventory.rst.inc

```diff
@@ -228,13 +228,99 @@ RECORD_TIME_OUT_OF_RANGE
 
 
 
-2.1. ParticipantErrorGroup / IndexErrors
+2.1. ParticipantErrorGroup / CommonErrors
+===================================================================================================================
+
+Common errors raised in Daml services and components.
+
+
+
+.. _error_code_REQUEST_TIME_OUT:
+
+REQUEST_TIME_OUT
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: This rejection is given when a request processing status is not known and a time-out is reached.
+
+    **Category**: DeadlineExceededRequestStateUnknown
+
+    **Conveyance**: This error is logged with log-level INFO on the server side and exposed on the API with grpc-status DEADLINE_EXCEEDED including a detailed error message.
+
+    **Resolution**: Retry for transient problems. If non-transient contact the operator as the time-out limit might be too short.
+
+
+
+
+.. _error_code_SERVER_IS_SHUTTING_DOWN:
+
+SERVER_IS_SHUTTING_DOWN
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: This rejection is given when the participant server is shutting down.
+
+    **Category**: TransientServerFailure
+
+    **Conveyance**: This error is logged with log-level INFO on the server side and exposed on the API with grpc-status UNAVAILABLE including a detailed error message.
+
+    **Resolution**: Contact the participant operator.
+
+
+
+
+.. _error_code_SERVICE_INTERNAL_ERROR:
+
+SERVICE_INTERNAL_ERROR
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: This error occurs if one of the services encountered an unexpected exception.
+
+    **Category**: SystemInternalAssumptionViolated
+
+    **Conveyance**: This error is logged with log-level ERROR on the server side. It is exposed on the API with grpc-status INTERNAL without any details for security reasons.
+
+    **Resolution**: Contact support.
```

Comment:
New INTERNAL error, which is more generic than the `LEDGER_API_INTERNAL_ERRO`

Reply:
This is changing the behavior of the places in code that use `LEDGER_API_INTERNAL_ERROR` to `SERVICE_INTERNAL_ERROR`. As far as I can tell, this affects ledger api client code only, but I may be wrong about that. As a minimum, this needs to be documented in the release notes.

Reply:
Not all of them change to `SERVICE_INTERNAL_ERROR`, only the ones in the Ledger API client code (which I think is fine imho - error codes are not forwarded over the wire anyway for INTERNALs).

Reply:
I know that it's deprecated, but added it to the changelog. See ⬆️